### PR TITLE
docs(rxMultiSelect): FRMW-295 Migrate rxMultiSelect

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-295

- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

**Comment by 100stacks** [comment]
LGTM

File: src/rxMultiSelect/README.md [hunk 1]
```diff
@@ -2,38 +2,4 @@
 
 [API Documentation](ngdocs/index.html#/api/rxMultiSelect)
```
Comment (CITguy) [inline src/rxMultiSelect/README.md]: Please remove "API Documentation" link.

File: src/rxMultiSelect/rxMultiSelect.js [hunk 2]
```diff
@@ -21,7 +23,36 @@ angular.module('encore.ui.rxMultiSelect', ['encore.ui.rxSelectFilter'])
...
+ *
+ * The preview text (what is shown when the element is not active) follows the following rules:
+ * * If no items are selected, show "None".
+ * * If only one item is selected from the dropdown, its label will display.
+ * * If > 1 but < n-1 items are selected, show "[#] Selected".
+ * * If all but one item is selected, show "All except [x]"
+ * * If all items are selected, show "All Selected".
  *
  * @param {String} ng-model The scope property that stores the value of the input
  * @param {Array} [options] A list of the options for the dropdown
```
Comment (CITguy) [inline src/rxMultiSelect/rxMultiSelect.js]: `options` is optional. Please update param signature to read

```
@param {Array=} options A list of the options for the dropdown.
```

**Comment by ssalinas24** [comment]
@CITguy Thanks for the feedback. I have made the changes in my second commit.

**Comment by CITguy** [comment]
LGTM

**Pull request merged**